src/lib.rs

```diff
@@ -208,8 +256,17 @@ fn catalog_impls_send_sync() {
 #[test]
 fn catalog_insert() {
     let mut cat = Catalog::new();
-    cat.insert(Message::new("thisisid", None, vec![]));
-    cat.insert(Message::new("anotherid", Some("context"), vec![]));
+    cat.insert(Message::new(
+        "thisisid",
+        None,
+        vec![],
+    ));
+    cat.insert(Message::with_plural(
+        "anotherid",
+        Some("context"),
+        vec![],
+		Some("thisispluralid")
+    ));
```

Comment:
This test should split this out, so that we have one each of singular, singular with context, plural, plural with context.

Reply:
@rmccue This has been resolved.